src/service_args.py

```diff
@@ -144,3 +146,13 @@ def __crust(self):
             self.__set_chain_name('maxwell', 0)
         elif self.chain_name == 'crust-rocky':
             self.__set_chain_name('rocky', 0)
+
+    def __sora(self):
+        '''
+        Everything after 'sora-' will be used as the chain name.
+        Examples:
+        --chain=sora-main -> --chain=main
+        --chain=sora-another-network -> --chain=another-network
+        '''
+        chain_name = self.chain_name.split('-', 1)[1]
+        self.__set_chain_name(chain_name, 0)
```

Comment:
This is an option to make it future proof for other networks. (as long as the docker image does not change).